src/main/resources/application.conf

```diff
@@ -36,6 +36,12 @@ workflow-options {
 
   // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
+  workflow-log-dir: "cromwell-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
+  workflow-log-temporary: true
```

Comment:
Fwiw I know people who refuse to use software which writes files they didn't request due to them supposedly being a Trojan. There's no one right answer.

Personally I don't like stuff leaving detritus I don't know about but I wouldn't call something a Trojan for doing so :)